src/pages/PageCallManage/index.tsx

```diff
@@ -87,163 +187,124 @@ class PageCallManage extends React.Component<{
     )
   }
 
-  renderCall = (c?: any, isVideoEnabled?: boolean) => {
-    if (!c) {
+  renderCall = (currentCall?: any, isVideoEnabled?: boolean) => {
```

Comment:
do not use type any, if it's not mandatory to use it

Reply:
If any is not used, VS code validator was showing error